src/app/demo/typescript/typescript.component.ts

```diff
@@ -10,6 +10,36 @@ import {ng2tsConfig} from '../../../../ng2ts/ng2ts';
 })
 
 export class TypescriptComponent {
+
+  introJsBefore() {
+    let squiggly;
+
+    squiggly = document.querySelector('.highlighted-code');
```

Comment:
VS code should be able to do this automatically. 
Or maybe consider moving to webstorm?